controllers/dscinitialization/dscinitialization_controller.go

```diff
@@ -83,6 +84,11 @@ func (r *DSCInitializationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	// Set platform
 	platform := currentOperatorRelease.Name
 
+	createUsergroup, err := cluster.IsDefaultAuthMethod(ctx, r.Client)
```

Comment:
nit: this can probably be moved closed where it is actually in use so i.e. if there's no DSCI, then there's no even need to retrieve the auth config. Same as if the reconcile request is not about the DSCI.